Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM golang
```

Comment:
I would use `FROM golang:1.15-alpine` to pin a specific version.

And I would `RUN apk add --no-cache ca-certificates` to make sure it can make outgoing https connections.